firefox-ios/Client/Frontend/Strings.swift

```diff
@@ -3731,6 +3731,21 @@ extension String {
         tableName: nil,
         value: "OK",
         comment: "OK button to dismiss the error prompt.")
+    public static let ScanQRCodeConfirmOpenURLMessage = MZLocalizedString(
+        key: "ScanQRCode.ConfirmOpenURL.Message.v100",
+        tableName: nil,
```

Comment:
@cyndichin I'm not super-familiar with our string workflow so wanted to double-check on this. Kept the `tableName` here nil for these new strings to match the others, wasn't sure whether there were broader implications for changing `tableName` for all of the strings in the struct? (Would that require updating the version number etc.?)

Reply:
Since these are new, we can update the new ones to follow our current policy and keep the old ones as-is. I would specify the `tableName`, which could be `ScanQRCode` and update the version number to `v129` since that's our current build version. 

Each string kind of acts independently, so should be safe to make the updates.

Reply:
@cyndichin Will do, thank you